docs/deployment-process/issue-tracking/azure-devops.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Azure DevOps Work Item Tracking Integration
+description: Configure Azure DevOps work item tracking with Octopus.
+position: 40
+---
+
+**Octopus 2019.7.3** introduced support to integrate Octopus with Azure DevOps work items. The integration includes the ability to:
+- Automatically add links to Azure DevOps work items from releases and deployments in Octopus.
+- Retrieve release notes from Azure DevOps work item comments to assist in automating release note generation.
+
+## How Azure DevOps Integration Works
+
+![Octopus Azure DevOps integration - how it works diagram](images/octo-azure-devops-how-it-works.png "width=500")
+
+1. Associate code changes with their relevant work items in any of the following ways:
+    - Edit a pull request in Azure DevOps, and use the **Work Items** panel to select a work item.
+    - Edit a work item in Azure DevOps, and use the **Development** panel to add a pull request link (before build), or a commit link, or a build link.
+    - When you commit code, include "#" followed by a valid work item ID in the commit message. *Note: Requires first enabling the repository setting "Automatically create links for work items mentioned in a commit comment" under Project Settings (Repositories).*
```

Comment:
This looks a bit messy. It might be better to add the note on a new line? @wordlee can comment on structure better though. :)